packages/rocketchat-importer-hipchat-enterprise/server/importer.js

```diff
@@ -108,12 +125,14 @@ export class HipChatEnterpriseImporter extends Base {
 										ts: new Date(m.UserMessage.timestamp.split(' ')[0]),
 									});
 								} else if (m.NotificationMessage) {
+									const text = m.NotificationMessage.message.indexOf('/me ') === -1 ? m.NotificationMessage.message : `${ m.NotificationMessage.message.replace(/\/me /, '_') }_`;
```

Comment:
this will replace any `/me` text and not only the slash command (which starts with `/me`) but since this what was done before, we can improve it later